.env

```diff
@@ -1 +1,5 @@
 SASS_PATH=./src/styles
+
+REACT_APP_OIDC_AUTHORITY="http://tunnistamo-backend:8000"
```

Comment:
I recommend a solution that doesn't require a local tunnistamo by default. In kukkuu we have an .env.development like this: 

```REACT_APP_API_URI="https://kukkuu.test.kuva.hel.ninja/graphql"
REACT_APP_OIDC_AUTHORITY="https://tunnistamo.test.kuva.hel.ninja"```

And if we want to run it against a local tunnistamo we create a `.env.development.local` with the local overrides